### PR TITLE
ci.yml: changes

### PR DESCRIPTION
**Pull request by Margen67**

### CODE REVIEW

**Comment by aspargas2** [comment]
One problem with this: because the devkitarm docker image comes with an older version of git (or at least I think that's why), checkout v2 can't properly clone the repo and instead does it with github's rest API, which leaves out the entire .git folder. This in turn prevents gm9's Makefile from having any information on what commit is being built, so the resulting build artifact will be missing everything derived from that (this can be seen in the artifact produced by the CI build run on this pull request). 

Other than that, these changes are fine. I would recommend simply leaving it using checkout v1, as I do no know of any downside to doing so.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -5,10 +5,10 @@ on: [push, pull_request]
 jobs:
   build:
     runs-on: ubuntu-18.04
-    container: devkitpro/devkitarm:latest
+    container: devkitpro/devkitarm
 
     steps:
-      - uses: actions/checkout@v1
```
Comment (d0k3) [inline .github/workflows/ci.yml:11]: Leave this at checkout v1 - this will make things easier for us. See @aspargas2's comment.

**Review by d0k3 — CHANGES_REQUESTED** [review]
One small change needed, then we can merge this.

**Comment by Margen67** [comment]
> One problem with this: because the devkitarm docker image comes with an older version of git (or at least I think that's why), checkout v2 can't properly clone the repo and instead does it with github's rest API, which leaves out the entire .git folder. This in turn prevents gm9's Makefile from having any information on what commit is being built, so the resulting build artifact will be missing everything derived from that (this can be seen in the artifact produced by the CI build run on this pull request).
> 
> Other than that, these changes are fine. I would recommend simply leaving it using checkout v1, as I do no know of any downside to doing so.

@aspargas2 Checkout v2 does a [shallow clone by default](https://github.com/actions/checkout#fetch-all-history-for-all-tags-and-branches):
```
# Number of commits to fetch. 0 indicates all history for all branches and tags.
# Default: 1
fetch-depth: ''
```

**Comment by Margen67** [comment]
> 
> 
> One small change needed, then we can merge this.

@d0k3 Fixed.

**Review by d0k3 — APPROVED** [review]
Looks good to me!

**Pull request merged**